CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Forge
 
+#### Added
+
+- support for `deploy_syscall` directly in the test code (alternative to `deploy`)
```

Comment:
```suggestion
- support for `deploy_syscall` directly in the test code (alternative to `deploy` cheatcode)
```

Reply:
`deploy` is not a cheatcode actually since it does not cheat anything